packages/eas-cli/src/commands/build/delete.ts

```diff
@@ -0,0 +1,122 @@
+import gql from 'graphql-tag';
+
+import EasCommand from '../../commandUtils/EasCommand';
+import { ensureBuildExistsAsync, fetchBuildsAsync, formatBuild } from '../../commandUtils/builds';
+import { ExpoGraphqlClient } from '../../commandUtils/context/contextUtils/createGraphqlClient';
+import { EASNonInteractiveFlag } from '../../commandUtils/flags';
+import { withErrorHandlingAsync } from '../../graphql/client';
+import { Build, DeleteBuildMutation, DeleteBuildMutationVariables } from '../../graphql/generated';
+import Log from '../../log';
+import { ora } from '../../ora';
+import { getDisplayNameForProjectIdAsync } from '../../project/projectUtils';
+import { confirmAsync, selectAsync } from '../../prompts';
+
+async function deleteBuildAsync(
+  graphqlClient: ExpoGraphqlClient,
+  buildId: string
+): Promise<Pick<Build, 'id'>> {
+  const data = await withErrorHandlingAsync(
+    graphqlClient
+      .mutation<DeleteBuildMutation, DeleteBuildMutationVariables>(
+        gql`
+          mutation DeleteBuildMutation($buildId: ID!) {
+            build(buildId: $buildId) {
+              deleteBuild(buildId: $buildId) {
+                id
+              }
+            }
+          }
+        `,
+        { buildId }
+      )
+      .toPromise()
+  );
+  return data.build!.deleteBuild;
+}
+
+export async function selectBuildToDeleteAsync(
+  graphqlClient: ExpoGraphqlClient,
+  projectId: string,
+  projectDisplayName: string
+): Promise<string | null> {
+  const spinner = ora().start('Fetching builds…');
+
+  let builds;
+  try {
+    builds = await fetchBuildsAsync(graphqlClient, projectId);
+    spinner.stop();
+  } catch (error) {
+    spinner.fail(
+      `Something went wrong and we couldn't fetch the builds for the project ${projectDisplayName}.`
+    );
+    throw error;
+  }
+  if (builds.length === 0) {
+    Log.warn(`There aren't any builds for the project ${projectDisplayName}.`);
```

Comment:
```suggestion
    Log.warn(`There are no builds for the project ${projectDisplayName}.`);
```
Or maybe something like "we couldn't find any builds"? I think it feels more human?

Reply:
The message was copied from an existing command, but I can update both 😄

Reply:
Updated